File: android/src/main/java/com/nateshmbhat/card_scanner/logger/Logger.kt
package com.nateshmbhat.card_scanner.logger

import android.util.Log
import com.nateshmbhat.card_scanner.scanner_core.models.CardScannerOptions

fun debugLog(message: String, scannerOptions: CardScannerOptions, tag: String = "card_scanner_debug_log") {
  if (scannerOptions.enableDebugLogs) {
    Log.d(tag, message)
  }
}

File: ios/Classes/Logger/Logger.swift
//
//  Logger.swift
//  card_scanner
//
//  Created by Mohammed Sadiq on 13/02/21.
//

import Foundation

func debugLog(_ data: String, scannerOptions: CardScannerOptions) {
    if scannerOptions.enableDebugLogs {
        NSLog(data)
    }
}
